crates/rpc/src/types/id.rs

```diff
@@ -8,6 +8,7 @@ use serde::{Deserialize, Serialize};
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
 #[serde(untagged)]
 pub enum ID {
+    Named(String),
```

Comment:
```suggestion
    Head,
    Justified,
    Finalized,
    Genesis,
```

Instead of named it should be like this, except sorted in alphabetical order with the rest of the enums options